doc/General/Callback-Stats-and-Privacy.md

```diff
@@ -28,7 +28,7 @@ Now onto the bit you're interested in, what is submitted and what we do with tha
 - Q. How often is data submitted? A. We submit the data once a day according to running daily.sh via cron. If you disable this then opting in will not have any affect.
 - Q. Where can I see the data I submitted? A. You can't see the data raw, but we collate all of the data together and provide a dynamic site so you can see the results of all contributed stats [here](https://stats.librenms.org)
 - Q. I want my data removed. A. That's easy, simply press 'Clear remote stats' in the About LibreNMS page of your control panel, the next time the call back script is run it will remove all the data we have.
-- Q. I clicked the 'Clear remote stats' button by accident. A. No problem, before daily.sh runs again - just opt back in, all of your existing data will stay.
```

Comment:
A. means Answer, check the other ones.

Reply:
yeah  i know just noticed that. :(

Reply:
@kkrumm1 you making that mistake indicates to me that it could use some improvement.  Perhaps bolding Q. and A.

Reply:
Okay yeah, I could see that helping if a user came in and was trying to read that. 
shall I re-open it?

Reply:
up to you, whatever is easier.